### ISSUE TITLE
Cannot build Cap3 project when this plugin is installed

### ISSUE THREAD

**Issue by kozakhs** [capacitor-community/native-market#14]
**Describe the bug**
When this command is executed
ionic capacitor run android --livereload --external --livereload-url=http://192.168.2.147:8100

it results in this error:
```
...
NativeMarket.java:21: error: cannot access androidx.appcompat.app.AppCompatActivity
[capacitor]         Context context = this.bridge.getActivity().getApplicationContext();
[capacitor]         ^
[capacitor]         class file for androidx.appcompat.app.AppCompatActivity not found
....
```

**To Reproduce**
1) Install latest version of capacitor3 via npm
2) Install this plugin via npm install @capacitor-community/native-market
3) Run ionic cap sync
4) Try to run the app (on external device)

**Expected behavior**
Run the app.

**Desktop (please complete the following information):**
 - OS Win 10

**Smartphone (please complete the following information):**
 - Samsung Galaxy A40, Android 11

**Comment by kozakhs** [comment]
Any response please?

**Comment by hermitdemschoenenleben** [comment]
this fork works for me with capacitor 3: https://github.com/hermitdemschoenenleben/native-market

**Comment by riderx** [comment]
@hermitdemschoenenleben can you publish it on npm ?

**Comment by riderx** [comment]
if you need i published it with capacitor 3 https://github.com/Cap-go/native-market

**Comment by jcesarmobile** [comment]
I've released 4.0.0 with Capacitor 4 support, it should have fixed this issue.
Also 5.0.0 with Capacitor 5 support.
If the problem is still present, please, create a new issue and provide a sample app that reproduces the issue

There won't be a release with Capacitor 3 support.
